go.mod

```diff
@@ -99,7 +99,7 @@ require (
 	github.com/pmezard/go-difflib v1.0.0
 	github.com/pquerna/ffjson v0.0.0-20180717144149-af8b230fcd20 // indirect
 	github.com/prometheus/client_golang v1.0.0
-	github.com/prometheus/client_model v0.1.0
+	github.com/prometheus/client_model v0.0.0-20190812154241-14fe0d1b01d4
```

Comment:
Is there a reason to do this downgrade in 506ddeeb70ed6b904074f730376bf10a5b617155? The diff inside the lib https://github.com/prometheus/client_model/compare/14fe0d1b01d4...v0.1.0 shows that there are no go related changes, other than the fact that they have removed protobuf-stuff for other languages in `v0.1.0`.

So in fact this will just revert @dims PR https://github.com/kubernetes/kubernetes/pull/87093.

Haven't tested it, but if there is no difference I guess keeping the tagged version would be preferred?